lib/experimental/Widgets/Layout/Dashboard/index.tsx

```diff
@@ -5,18 +5,31 @@ import {
   forwardRef,
   ReactNode,
   useEffect,
+  useMemo,
   useRef,
   useState,
 } from "react"
 import { Masonry } from "react-masonry"
 import { Widget } from "../../Widget"
 
 type DashboardProps = {
+  widgetWidth?: "sm" | "md" | "lg"
   children?: ReactNode[]
 }
 
 const _Dashboard = forwardRef<HTMLDivElement, DashboardProps>(
-  function Dashboard({ children }, ref) {
+  function Dashboard({ children, widgetWidth = "sm" }, ref) {
+    const maxWidgetWidth = useMemo(() => {
+      switch (widgetWidth) {
+        case "sm":
+          return 340
+        case "md":
+          return 480
+        case "lg":
+          return 640
+      }
+    }, [widgetWidth])
```

Comment:
Sorry, was just reviewing this and I approved without thinking. Why do we need a `useMemo` for this? It's not an expensive computation.

Reply:
Because is used in the useEffect below, without useMemo, the useEffect will execute in every render

Reply:
That's not the case! The hook will be checked by value, and an integer is a stable value.

Reply:
mmm... true!

Reply:
True is also a stable value 😆

Reply:
let me do a quick PR to make these changes